content/pt-br/contributor-ladder/_index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Escala de Contribuição
+toc_hide: true
+menu:
+  main:
+    weight: 10
+    pre: <i class='fas fa-pen-square'></i>
+---
+
+Olá! 👋 Obrigado pelo seu interesse em contribuir com o projeto CNCF Glossário Cloud Native. Seja contribuindo com novos termos, ajudando a localizar o glossário em seu idioma nativo ou ajudando outras pessoas a começar, há muitas maneiras de se tornar um membro ativo desta comunidade. Este documento descreve as diferentes funções do colaborador no projeto e as responsabilidades e privilégios que as acompanham.
+
+## 1. Contribuidores
+
+O glossário é para todos. Qualquer pessoa pode se tornar um contribuidor simplesmente contribuindo com o projeto.
+É esperado que todos os contribuidores sigam o [código de conduta da CNCF](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
+
+Existem várias formas de você contribuir com o projeto, incluindo:
+
+- **Contribuidores de conteúdo**: qualquer pessoa que melhore os termos existentes ou contribua com novos,
+- **Contribuidores de localizações**: qualquer pessoa que ajude a localizar o conteúdo para outro idioma,
+- **Voluntários**: qualquer pessoa que ajude os outros no Github, Slack ou qualquer membro da comunidade que precise de ajuda,
+- **Embaixadores**: qualquer pessoa que ajude a divulgar o projeto e ajudar a a contribuir.
+
+Contribuidores podem ter múltiplas funções ou focar em uma única área. **Todas as contribuições são igualmente importantes** e ajudam no crescimento da comunidade. Para contribuições em conteúdo e localização, veja o [guia de contribuição](https://glossary.cncf.io/pt-br/contribute/) e o [guia de estilo](https://glossary.cncf.io/pt-br/style-guide/).
+
+## 2. Aprovadores
+
+Aprovadores são responsáveis por prover feedbacks e aprovar os PRs (pull requests). Um contribuidor ativo pode se tornar um aprovador (veja "Se tornando um aprovador"). O glossário possui dois tipos de aprovadores: (1) aprovadores para o glossário em inglês e (2) aprovadores para os times de localização.
+
+É esperado que os aprovadores do glossário:
+
+- Revisem os PRs em uma perspectiva técnica,
+- Atribuiam as *issues* aos contribuidores e apliquem as *labels* corretamente,
+- Ajudem os contribuidores com feedback e toda a ajuda que precisem.
+
+Se um aprovador não estiver mais interessado ou não consegue realizar as atividades acima, é importante que comunique aos mantenedores para que seja formalizada a saída da função.
+
+## Aprovadores do Glossário em Inglês
```

Comment:
```suggestion
### Aprovadores do Glossário em Inglês
```

as per https://glossary.cncf.io/contributor-ladder/#english-glossary-approvers and https://raw.githubusercontent.com/cncf/glossary/main/content/en/contributor-ladder/_index.md